Doc/c-api/bool.rst

```diff
@@ -19,29 +19,30 @@ are available, however.
 
 .. c:var:: PyObject* Py_False
 
-   The Python ``False`` object.  This object has no methods.  It needs to be
-   treated just like any other object with respect to reference counts.
+   The Python ``False`` object.  This object has no methods and is immortal.
+
+.. versionchanged:: 3.12
+   :const:`Py_False` is immortal.
```

Comment:
`:c:const:` maybe?

Reply:
I think I picked up ``:const:`` from the Py_RETURN_FALSE doc. 

Is there such a thing as ``:c:const:`` (grep on the docs dir doesn't give me any).

Reply:
I think we actually want ":c:data:". That's how the link to Py_None in https://docs.python.org/3.10/c-api/slice.html#c.Py_Ellipsis is created.

Reply:
I had :c:data: initially but there was some build error. I'll try it again.